packages/playground-ui/src/index.css

```
@@ -343,11 +391,11 @@ html.light {
   --shadow-inner: inset 0 2px 4px oklch(0 0 0 / 0.3);
   --shadow-card: 0 2px 8px oklch(0 0 0 / 0.4);
   --shadow-elevated: 0 8px 24px oklch(0 0 0 / 0.5);
-  /* Inset top highlight (gloss) instead of outer 1px ring — outer ring stacked with the
-     explicit `border` on every consumer (Dialog, Select/Dropdown content, etc.) and read as
-     a duplicated outline. */
-  --shadow-dialog:
-    inset 0 1px 0 0 oklch(1 0 0 / 0.04), 0 8px 24px -8px oklch(0 0 0 / 0.28), 0 24px 56px -16px oklch(0 0 0 / 0.36);
+  /* Drop-shadow only. A previous version added `inset 0 1px 0 0 oklch(1 0 0 / 0.04)` for
+     a 1px gloss highlight on top, but Tailwind v4 inlines @theme values into utility
+     classes so it can't be themed per-mode via a CSS var override — and the white inset
+     reads as a dirty band on dark surfaces and is invisible on white-ish light surfaces. */
```

Stylelint (17.11.0)

[error] Expected empty line before comment (comment-empty-line-before)

(comment-empty-line-before)